extensions/functions_arithmetic.yaml

```diff
@@ -175,12 +175,20 @@ scalar_functions:
         return: fp64
   -
     name: "divide"
-    description: "Divide one value by another. Partial values are truncated."
+    description: >
+      Divide x by y. In the case of integer division, partial values are truncated.
+      The `on_division_by_zero` option governs behaviour in cases where y is 0 and x is not 0.
+      The `on_domain_error` option governs behaviour for 0/0 and Inf/Inf etc.
```

Comment:
It looks fine to me, but cc @pitrou 

PS. I honestly can't tell if github actually added this comment when I first reviewed this because the UI was glitching all to hell, but it seems to be missing for me now, so... Please excuse me if this ends up being duplicated.

Reply:
That makes sense, updated now!